packages/main/src/components/AnalyticalTable/demo/demo.stories.tsx

```diff
@@ -56,39 +61,38 @@ const columns = [
 const data = generateData(200);
 const dataTree = generateData(20, true);
 
-export const defaultTable = () => {
+const renderTable = () => {
```

Comment:
can you please export this story again and remove the new export you have creating? having these nested component demos is breaking our storybook:
![image](https://user-images.githubusercontent.com/8775211/73559149-60581080-4409-11ea-97b9-1ce2453e8f41.png)